### DISCUSSION TITLE
Relation manager using resource not form and table

### DISCUSSION THREAD

**Discussion by oskobri** [filamentphp/filament#4862] [Help] [answered]
Hello everyone,

I'm using laravel nova for my project but i want to explore new possibilities.
I'm trying to migrate from nova to filament to know if all i can do with nova can be possible with filament. 
I don't understand the relation manager. Why for hasMany or BelongsToMany i need to recreate form and table inside the relation manager instead of using the resource itself ?

**Comment by danharrin** [comment, marked as answer]
You can just call `ResourceName::form($form)`?

**Reply by danharrin** [reply]
But the reason that they are different is because you will often want limited functionality in the relation manager compared to the resource, to reduce complexity inside of the modal

**Reply by oskobri** [reply]
Thanks for the reply. Indeed it's working with `ResourceName::form($form)` but the actions on the table open a modal and not a page. Is there a way to open the resource in a page ? I need it because the hasMany resource will not be accessible via the navigation but only from this table. It will contain too many information and relations to be opened inside a modal. In addition to that, the translated attributes are not working inside a modal, I've got an object inside my text input. It's only working on a page because the translatable trait is on the page class.

**Reply by danharrin** [reply]
https://filamentphp.com/tricks/relation-manager-edit-record-without-modal

If you remove the modals you wont even need to define a `form()` on the relation manager

**Reply by oskobri** [reply]
OK thanks !
